flowzone.yml

```
@@ -418,12 +437,12 @@
   - &rejectSelfHostedRunners
     name: Reject self-hosted runners
     if: |
+      steps.is_approved_external_pr.outputs.result == 'false' &&
       github.event.pull_request.head.repo.full_name != github.repository &&
       (
-        contains(fromJSON(inputs.runs_on), 'self-hosted') ||
-        contains(fromJSON(inputs.custom_runs_on), 'self-hosted') ||
-        contains(fromJSON(inputs.custom_runs_on).*, 'self-hosted') ||
-        contains(fromJSON(inputs.docker_runs_on).*, 'self-hosted')
+        contains(inputs.runs_on, 'self-hosted') ||
+        contains(inputs.custom_runs_on, 'self-hosted') ||
+        contains(inputs.docker_runs_on, 'self-hosted')
       )
     run: |
       echo "::error:External workflows can not be used with self-hosted runners. \
```

```suggestion
      echo "::error::External workflows can not be used with self-hosted runners. \
```
We should fix this typo I made previously. It's preventing the error from showing in the summary.

---

Same typo is on line 423 and 433 if you could fix any instances of `::error:` with `::error::` that would be great